src/main/java/de/srendi/advancedperipherals/common/util/StatusConstants.java

```diff
@@ -0,0 +1,42 @@
+package de.srendi.advancedperipherals.common.util;
+
+/**
+ * A collection of constants used as return types for several peripherals
+ */
+public enum StatusConstants {
+
+    // Crafting Jobs
+    CALCULATION_STARTED,
+    CRAFTING_STARTED,
+    JOB_CANCELED,
+    JOB_DONE,
+    NOT_CRAFTABLE,
+    MISSING_ITEMS,
+    CPU_DOES_NOT_EXIST,
```

Comment:
```suggestion
    CPU_NOT_FOUND,
```

I feel "does" is too much here